README.md

```diff
@@ -1,7 +1,35 @@
-### Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors
+# Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors
 
 This paper is accepted to Findings of [ACL2023](https://aclanthology.org/2023.findings-acl.426/).
 
+## Getting Started
+
+This codebase is [available on pypi.org via](https://pypi.org/project/npc-gzip) 
+
+
+```bash
+
+pip install npc-gzip
+
+```
+
+See the [examples](./examples/imdb.py) directory for example usage.
+
+
+## Testing
+
+This package utilizes `poetry` to maintain its dependencies and `pytest` to execute tests. To get started running the tests:
+
+```bash
+
+poetry shell
+poetry install
```

Comment:
I recommend putting `poetry install` before `poetry shell`, rather than after.

Reply:
Shouldn't poetry shell go first since it creates the virtual environment that you would then want to `poetry install` into?

Reply:
Either command will create the environment if it doesn't exist yet. I prefer to run `poetry install` first, because when I've run `poetry shell` first I often find myself confused that something isn't working and it turns out the environment exists but the project isn't installed, and because the purpose of `poetry shell` is to allow the installed project and dependencies to be used. (Running `poetry shell` and then `pytest` has a similar effect as what `poetry run pytest ...` achieves on CI.) But really both ways are guaranteed to work, so if you prefer to have `poetry shell` first, that's definitely fine.

Reply:
Let's leave it as is, if it proves problematic in the future we know the quick fix.